t/60leaks.t

```diff
@@ -7,6 +7,8 @@ use vars qw($test_dsn $test_user $test_password);
 use lib 't', '.';
 require 'lib.pl';
 
+use constant SHOW_PROGRESS => ($ENV{SHOW_PROGRESS} ? 1 : 0);
```

Comment:
note "You can set SHOW_PROGRESS environment variable to monitor the progress of slow tests."
        if ! $ENV{SHOW_PROGRESS} && $ENV{TEST_VERBOSE};